main/states/busy_state.py

```diff
@@ -49,7 +49,7 @@ def on_enter(self, payload=None):
             if 'volume' in reply.keys():
                 m = alsaaudio.Mixer()
                 m.setvolume(int(reply['volume']))
-                subprocess.call(['play', '{0} &'.format(self.components.config['detection_bell_sound'])])  # nosec #pylint-disable type: ignore
+                subprocess.call(['play', '{}'.format(self.components.config['detection_bell_sound'])])  # nosec #pylint-disable type: ignore
```

Comment:
If there is only `'{}'`, why do you have to write `'{}'.format(something)` to be long?

Reply:
@hongquan , i don't understand your query

Reply:
Instead of:

```py
'{}'.format(something)
```

Isn't it cleaner to write:

```py
something
```

Or if `something` is not a string, just write

```py
str(something)
```